xgcm/grid.py

```diff
@@ -1399,7 +1401,12 @@ def interp_like(self, array, like):
             except KeyError:
                 continue
             if position_like != position_array:
-                array = self.interp(array, axname)
+                array = self.interp(
+                    array,
+                    axname,
+                    fill_value=fill_value.get(axname, None),
+                    boundary=boundary.get(axname),
+                )
         return array
 
     def _interp_metric(self, da, axes):
```

Comment:
How do we handle boundaries here?

Reply:
Boundary can either be a string or dictionary, and the line at the beginning of the loop:
```python
boundary = self._parse_axes_kwargs(boundary)
```
will assign boundary values and then you can select per axis name inside the loop `boundary=boundary.get(axname)`